velox/connectors/hive/HiveDataSink.cpp

```diff
@@ -738,6 +739,11 @@ HiveDataSink::maybeCreateBucketSortWriter(
       sortCompareFlags_,
       sortPool,
       writerInfo_.back()->nonReclaimableSectionHolder.get(),
+      exec::PrefixSortConfig{
```

Comment:
I don't think this even works.

connectorQueryCtx_->sessionProperties() contains only connector-specific properties. It doesn't include all properties.

```
std::shared_ptr<connector::ConnectorQueryCtx>
OperatorCtx::createConnectorQueryCtx(
    const std::string& connectorId,
    const std::string& planNodeId,
    memory::MemoryPool* connectorPool,
    const common::SpillConfig* spillConfig) const {
  return std::make_shared<connector::ConnectorQueryCtx>(
      pool_,
      connectorPool,
      driverCtx_->task->queryCtx()->connectorSessionProperties(connectorId),
      spillConfig,
      std::make_unique<SimpleExpressionEvaluator>(
          execCtx()->queryCtx(), execCtx()->pool()),
      driverCtx_->task->queryCtx()->cache(),
      driverCtx_->task->queryCtx()->queryId(),
      taskId(),
      planNodeId,
      driverCtx_->driverId);
}
```

Reply:
@mbasmanova 
Sorry for the delay in updating for so long. Recently, my time has been occupied by some high-priority projects.

In that case, do you have any suggestions:
1. Inject configuration options into conncetorSession
2. Add a new interface configuration like spillConfig

I tend to inject it in the connectorSession, because after all, it is a configuration of algorithm details. It does not feel good to add a new configuration.

Reply:
@skadilover I think we can add PrefixSortConfig member variable to ConnectorQueryCtx. Currently, there is 'spillConfig()', we'll add 'prefixSortConfig()'.

Reply:
got it